vendor/assets/javascripts/u2f-api.js

```diff
@@ -0,0 +1,756 @@
+// Maintained at https://github.com/google/u2f-ref-code/blob/master/u2f-gae-demo/war/js/u2f-api.js
```

Comment:
is it possible to version/update this file so that we inherit changes?

Reply:
I don't believe this file has any formal management or distribution. I wish I did, and hunted a bit.

The [official docs](https://developers.yubico.com/U2F/Libraries/Using_a_library.html) under "Client-side" say "the easiest way is to use the https://demo.yubico.com/js/u2f-api.js library" and simply link to that file dynamically. That seems less than ideal, and unnecessary. I managed to find the Google ref code which seems like the canonical repo where that file is maintained, so I copied from there and provided this attribution link.